Adyen/UI/Form/Items/Value Pickers/Abstract/BaseFormPickerItem.swift

```diff
@@ -31,25 +31,40 @@ public struct BasePickerElement<ElementType: CustomStringConvertible>: PickerEle
     }
 
     public var description: String { element.description }
```

Comment:
also no documentation here :-)

Reply:
I think since the struct is `nodoc`, inside elements are ok to not contain nodoc etc?